crates/bevy_physics/src/d3/intersectors.rs

```diff
@@ -0,0 +1,94 @@
+use super::{geometry::*, ray::*};
+use glam::Vec3;
+
+pub struct RayHit {
+    t: f32,
```

Comment:
Please consider renaming it to something more meaningful.

Reply:
That's a good idea!